From 9d0b4a40e148e4bf13e3015f13f51d204b852e24 Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Thu, 11 Oct 2018 18:26:51 +0700
Subject: [PATCH 697/909] Revert "arm64: dts: renesas: r8a7795: Connect SATA to
 IPMMU-HC"

commit b76f977b7811515efe60b5daed3cfab13dc6dbc9 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit bceb2536860ac574dc7994a5d890dd8a2bccd149
to keep IPMMU address translation for SATA is disabled by default,
maintain the same IPMMU support environment with previous
released R-Car Linux BSP.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 6cdc57fc12eb..43f9295d3a51 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -2895,7 +2895,6 @@
 			power-domains = <&sysc R8A7795_PD_ALWAYS_ON>;
 			resets = <&cpg 815>;
 			status = "disabled";
-			iommus = <&ipmmu_hc 2>;
 		};
 
 		gic: interrupt-controller@f1010000 {
-- 
2.17.1

